<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="InputMqttClient" name="input_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="send_input_1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Value</string>
    </property>
   </widget>
   <widget class="InputMqttClient" name="input_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="send_input_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_input_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Topic: A/topic1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_input_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Topic: B/topic2</string>
    </property>
   </widget>
   <widget class="OutputMqttClient" name="output_1">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="OutputMqttClient" name="output_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_input_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>On A/topic1: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_input_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>On A/topic1 &amp; B/topic2: </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InputMqttClient</class>
   <extends>QLineEdit</extends>
   <header location="global">inputmqttclient.h</header>
  </customwidget>
  <customwidget>
   <class>OutputMqttClient</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">outputmqttclient.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
